specification/_json_spec/synonym_rule.get.json

```diff
@@ -0,0 +1,33 @@
+{
+  "synonym_rule.get": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/get-synonym-rule.html",
+      "description": "Retrieves a synonym rule from a synonym set"
+    },
+    "stability": "experimental",
+    "visibility": "public",
+    "feature_flag": "es.synonyms_api_feature_flag_enabled",
```

Comment:
I think we want to remove the FF from this file

Reply:
Yes, thanks for the catch! 👍